index.html

```diff
@@ -9,691 +9,80 @@
     dt {color: black}
     </style>
       <link href="https://www.w3.org/StyleSheets/TR/2016/W3C-ED" rel="stylesheet" type="text/css">
+    <script src="http://dev.w3.org/2009/dap/ReSpec.js/js/respec.js" class="remove"></script>
+    <script class="remove">
+      var respecConfig = {
+          specStatus: "ED",
+	  shortName: "PNG",
+	  subtitle: "Information technology — Computer graphics and image processing — Portable Network Graphics (PNG): Functional specification. ISO/IEC 15948:2003 (E)",
+	  copyrightStart: "1996",
+	  previousPublishDate: "2003-11-10",
+	  previousMaturity: "REC",
+	  edDraftURI: "https://github.com/w3c/PNG-spec",
+	  extraCSS: ["http://dev.w3.org/2009/dap/ReSpec.js/css/respec.css"],
+	  editors: [
+	      { name: "Chris Blume", url: "https://www.programmax.net", w3cid: 127631 },
+	      { name: "Pierre-Anthony Lemieux", company: "Sandflow Consulting LLC", companyURL: "https://www.sandflow.com" },
+	      { name: "Chris Lilley", url: "https://svgees.us", company: "W3C", companyURL: "https://www.w3.org" },
+	      { name: "Leonard Rosenthol", company: "Adobe Inc.", companyURL: "https://www.adobe.com" },
+	      { name: "Chris Seeger", company: "Comcast Corporation", companyURL: "https://www.comcast.com" }
```

Comment:
@digitaltvguy Does this information look correct? Do you have a personal website you want to list in addition to your company website? Also, can you go to https://www.w3.org/users/myprofile and get your W3C ID from the address bar once you log in?

Reply:
I updated my W3C page.  Can the company be listed as "Comcast Corp./NBCUniversal, LLC".  Thx

Reply:
I will update the company listing.
But what I meant was to tell me your W3C ID so I can include it.

Once you log in, it will redirect you to a page with a URL like https://www.w3.org/users/12345 . I need that number.

Reply:
oops!  
https://www.w3.org/users/125844